examples/simple_index.html

```diff
@@ -13,10 +13,13 @@ <h1>Nothing to see here, open a debug console in your browser to see the interes
 
     <script>
       // Input string (English and German mixed).
-      let input_value = 'Mo-Fr 10:00-20:00; Samstag und Sonntag geschlossen';
+      let input_value = 'Mo-Tu 10:00-20:00; Mi-Fr 10:00-20:00; Samstag und Sonntag geschlossen';
 
-      // Create opening_hours object (and output warnings in English).
-      let oh = new opening_hours(input_value, {}, { 'locale': 'en' });
+      // Language for the warnings (get it from the browser settings).
+      let warnings_language = navigator.language.split('-')[0];
```

Comment:
Can you try what happens when you pass the full locale string? We might want to support those language variants in the futures and if it gets passed like this, we cannot easily do that. If not supported, bonus points for adding support for it to the library. Also, I find `warnings_language` to specific.

Reply:
Yes, this "split approach" is not ideal. Support of full locale strings would be very nice, but I don't yet fully understand the translation process here. Without that "split approach" Austrian users for example would get english warnings instead of german. 

- browser language: de -> german warnings 👍
- browser language: en -> english warnings 👍
- browser language: de-AT -> english warnings 👎

Reply:
I also don’t have the full context loaded. It is acceptable to me when you just add the split workaround to the library itself.

Reply:
> Also, I find `warnings_language` to specific.

You're right. There are also errors. `errors_warnings_language` seems to long and `language` I find to unspecific. So maybe `hint_language`? And maybe also locale instead of language?

Reply:
Keep it simple. Just `locale` to match the [library API](https://github.com/opening-hours/opening_hours.js#library-api).

Reply:
Okay, I changed it to `locale`.